src/AppInstallerCLITests/Dependencies.cpp

```diff
@@ -206,4 +208,69 @@ TEST_CASE("DependencyNodeProcessor_NoMatches", "[dependencies]")
     REQUIRE(dependencyList.Size() == 0);
     REQUIRE(installOutput.str().find(Resource::LocString(Resource::String::DependenciesFlowNoMatches)) != std::string::npos);
     REQUIRE(result == DependencyNodeProcessorResult::Error);
-}
\ No newline at end of file
+}
+
+std::string GetCurrentProcessPathVariable()
+{
+    size_t requiredSize;
+    getenv_s(&requiredSize, nullptr, 0, "PATH");
+
+    if (requiredSize > 0)
+    {
+        char* buffer = new char[requiredSize];
+        errno_t errorResult = getenv_s(&requiredSize, buffer, requiredSize, "PATH");
```

Comment:
nit: almost always prefer the wide string version when calling OS apis

Reply:
changed to wide string version